src/ParkMate/ApplicationCore/ValueObjects/BookingPeriod.cs

```diff
@@ -11,28 +11,46 @@ public BookingPeriod(DateTime start, DateTime end)
         {
             Start = start;
             End = end;
+
+            if (start == end)
```

Comment:
Can be combined into a single if statement. Always use braces, even when optional. Constructor assignments and logic should come after the guard clause. 
```suggestion
            if (end <= start)
            {
                throw  new ArgumentException("End cannot be before or equal to start time");
            }
            Start = start;
	    End = end;
```